cognite_toolkit/_cdf_tk/commands/_profile.py

```diff
@@ -185,6 +187,251 @@ def _as_cell(self, value: PendingCellValue) -> str | Spinner:
         return str(value)
 
 
+class ProfileAssetCommand(ProfileCommand):
+    def __init__(self, print_warning: bool = True, skip_tracking: bool = False, silent: bool = False) -> None:
+        super().__init__(print_warning, skip_tracking, silent)
+        self.table_title = "Asset Profile for Hierarchy"
+        self.hierarchy: str | None = None
+        self.aggregators: dict[str, MetadataAggregator] = {}
+
+    class Columns:
+        Resource = "Resource"
+        Count = "Count"
+        DataSets = "DataSet"
+        DataSetCount = "DataSet Count"
+        Transformations = "Transformation"
+        RawTable = "Raw Table"
+        RowCount = "Rows"
+        ColumnCount = "Columns"
+
+    is_dynamic_table = True
+    _index_split = "-"
```

Comment:
What is the behavior if a dataset/asset has "-" char? "-" are allowed for data set names/externalIds.

Reply:
This is a flaw. Good catch, will follow up on slack.